### ISSUE TITLE
Back to top button has insufficient color contrast on hover

### ISSUE THREAD

**Issue by thekaveman** [Office-of-Digital-Services/cdt-ods-disaster-recovery#442]
This appears to be coming directly from the State Web Template, I don't think we've modified the styles for this button at all.
